shared/js/apps_api/apps.js

```diff
@@ -59,7 +59,7 @@ function storageApps()
         contentType: "application/json; charset=utf-8",
         data: JSON.stringify(value)
       }).done(function(data, text) {
-        callback(true)
+        callback(data)
```

Comment:
return the new value of the object (not necessarily the same thing we saved eg. the updatedAt timestamp)